cmd/snap/cmd_info.go

```diff
@@ -462,17 +434,32 @@ func (x *infoCmd) Execute([]string) error {
 			continue
 		}
 
-		if tryDirect(w, snapName, x.Verbose, termWidth) {
-			noneOK = false
-			continue
-		}
-		remote, resInfo, _ := x.client.FindOne(snapName)
-		local, _, _ := x.client.Snap(snapName)
+		var direct, local, remote *client.Snap
```

Comment:
direct sounds too boolean too me,  though probably they all have the problem already?

should we bit the bullet and rename  to  diskSnap (or fsSnap),  localSnap and remoteSnap? could be a follow up (the diff is already painful)

Reply:
this needs addressing as well